src/middleware/rateLimiter.js

```diff
@@ -2,23 +2,14 @@ const rateLimit = require("express-rate-limit");
 
 
 // Rate limiting
-let limiter;
-if (process.env.NODE_ENV === "development") {
-  limiter = rateLimit({
-    windowMs: 1 * 60 * 1000, // 1 minute
-    max: 3, // limit each IP to 3 requests per windowMs
-    message: "Your limit exceeded",
-    standardHeaders: true, // Return rate limit info in the `RateLimit-*` headers
-    legacyHeaders: false, // Disable the `X-RateLimit-*` headers
-  });
-} else {
-  limiter = rateLimit({
+
+  const limiter = rateLimit({
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/armerillo/ql-task/pullRequest?prid=9975518)
```suggestion
const limiter = rateLimit({
```